build/cloudbuild-tf-apply.yaml

```diff
@@ -17,11 +17,11 @@ substitutions:
   _POLICY_REPO: '/workspace/policy-library' # add path to policies here https://github.com/forseti-security/policy-library/blob/master/docs/user_guide.md#how-to-use-terraform-validator
 steps:
 - id: 'setup'
-  name: gcr.io/$PROJECT_ID/terraform
+  name: $_DEFAULT_REGION-docker.pkg.dev/$PROJECT_ID/$_GAR_REPOSITORY/terraform
   entrypoint: /bin/bash
   args:
   - -c
-  - |
+  - |''
```

Comment:
Is this needed?

Reply:
Good catch! 
This line should not have been changed. We will revert this it.